listenbrainz/db/couchdb.py

```diff
@@ -130,6 +130,20 @@ def fetch_data(prefix: str, user_id: int):
     return None
 
 
+def fetch_exact_data(database: str, document_id: str):
```

Comment:
Good -- last time I looked at the couchdb code it wasn't very friendly for adding new types of data, so this stuff helps a lot.